### DISCUSSION TITLE
보투게더 홍보 URL 인증 게시판

### DISCUSSION THREAD

**Discussion by Gilpop8663** [woowacourse-teams/2023-votogether#617] [기획]
댓글로 달아주세요

**Comment by Gilpop8663** [comment]
[스펙업｜취업,대학생,인턴,대기업,공기업,GSAT,대외활동,공모전대표카페](https://cafe.naver.com/specup)

https://cafe.naver.com/specup?iframe_url_utf8=%2FArticleRead.nhn%253Fclubid%3D15754634%2526articleid%3D7348866

<img width="1138" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/9ce26b1d-3e3f-4fd9-9993-8cea3c0c4806">

**Comment by Gilpop8663** [comment]
[취업대학교★대기업,공기업,GSAT,NCS,인턴,승무원,자소서,면접대표카페](https://cafe.naver.com/bebettergirls)

https://cafe.naver.com/bebettergirls/2003824?boardType=L

<img width="1301" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/db2ea74e-481a-4b31-8796-e66c53673b61">

**Comment by Gilpop8663** [comment]
[공준모★공기업,채용,공공기관취업,NCS,자소서,인턴,공무직,한전대표카페](https://cafe.naver.com/studentstudyhard)

https://cafe.naver.com/studentstudyhard/2375737?boardType=L

<img width="1392" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/76eaa6dd-5c99-42df-8fd1-a92c4510db54">

**Comment by Gilpop8663** [comment]
[공취모★공기업NCS,공공기관취업,코레일채용,한국사,자소서카페대표카페](https://cafe.naver.com/dakchi)

https://cafe.naver.com/dakchi/2806583?boardType=L

<img width="1368" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/2cf6f9c8-91a1-4e4e-9ecf-8416208b1e70">

**Comment by Gilpop8663** [comment]
[간준모*간호사,취업,채용,자기소개서,면접,이력서,스펙,국가고시대표카페](https://cafe.naver.com/nursingstudies)

https://cafe.naver.com/nursingstudies/1313346?boardType=L

<img width="1285" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/fdabb0bf-3583-4955-8f3a-54687db7d3a9">

**Comment by jeomxon** [comment]
<img width="798" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/51906604/f4fff8dc-c287-4835-aa60-6a5282986efe">

**Comment by aiaiaiai1** [comment]
<img width="792" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/103165859/c803457b-5387-4948-8ead-1c36ea083653">

**Comment by chsua** [comment]
![스크린샷 2023-09-14 오후 4 28 06](https://github.com/woowacourse-teams/2023-votogether/assets/113416448/5637a593-3420-4dc9-ab74-49d7e4894721)
오픈채킹 홍보

**Comment by chsua** [comment]
![image](https://github.com/woowacourse-teams/2023-votogether/assets/113416448/52f1570f-8eaa-4ba4-bf9f-bb9392750a3a)
여기 내보내기 당했습니다! 주의!

**Comment by Gilpop8663** [comment]
중고나라
1초에 평균 4.5건의 신상품 등록! 찾는 그 제품, 중고나라에는 다 있다!

생활멤버 19,168,852명


https://cafe.naver.com/joonggonara/1012732749?boardType=L


<img width="1299" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/04f99284-e9c5-4eb9-974d-47698f6b6169">

**Comment by inyeong-kang** [comment]
https://everytime.kr/367450/v/316308867 
![image](https://github.com/woowacourse-teams/2023-votogether/assets/81199414/e2e23861-6c34-4600-889c-569a92ee0da3)

**Comment by inyeong-kang** [comment]
https://cafe.naver.com/dokchi/12385304
![image](https://github.com/woowacourse-teams/2023-votogether/assets/81199414/8184d62f-24b0-4a08-81a2-87fac38e65b2)

**Comment by chsua** [comment]
취업뽀개기 카페
![스크린샷 2023-09-14 오후 4 53 56](https://github.com/woowacourse-teams/2023-votogether/assets/113416448/2091102d-c77e-4760-9cbf-9c53da239fe4)

**Comment by inyeong-kang** [comment]
개발동아리 홍보
![image](https://github.com/woowacourse-teams/2023-votogether/assets/81199414/c5828825-5270-48a3-b7d2-4ed4b0542c7d)

**Comment by aiaiaiai1** [comment]
https://okky.kr/articles/1468930
<img width="1125" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/103165859/f92c0233-a3b4-4180-af76-04994188b7e6">

**Comment by aiaiaiai1** [comment]
https://careerly.co.kr/comments/90621?utm_campaign=self-share
<img width="1133" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/103165859/2c3bf191-63ef-4f29-9ee9-ad07c2732155">

**Comment by aiaiaiai1** [comment]
https://community.programmers.co.kr/talk/2656
<img width="1057" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/103165859/e14490a6-ff65-4460-b883-9bf849e36984">

**Comment by inyeong-kang** [comment]
인스타 계정이 없어서 동아리 슬랙채널에 다시 한번 홍보했어요
우테코 전체-잡담 채널에도 올렸습니다
![image](https://github.com/woowacourse-teams/2023-votogether/assets/81199414/4d2f3f82-11f8-4d90-9368-30a95c73feff)

**Comment by chsua** [comment]
<img width="373" alt="스크린샷 2023-09-21 오후 1 27 32" src="https://github.com/woowacourse-teams/2023-votogether/assets/113416448/83357466-a0a3-4a5a-85ef-89bcdb13284a">

**Comment by chsua** [comment]
![image](https://github.com/woowacourse-teams/2023-votogether/assets/113416448/1098e6e3-abf1-4bb4-bff0-edf20ac53623)

**Reply by chsua** [reply]
당근입니다!

**Comment by Gilpop8663** [comment]
게임 관련 홍보 올리는 트워터 계정입니다

https://twitter.com/GilGim2752

**Comment by inyeong-kang** [comment]
축구 커뮤니티 게시판에 올렸어요
https://soccerline.kr/board/17198657?searchWindow=&searchType=0&searchText=&categoryDepth01=1&page=0
![image](https://github.com/woowacourse-teams/2023-votogether/assets/81199414/d07bdf3e-b867-4b63-9690-da45adf6e9af)

**Comment by woo-chang** [comment]
부산대학교 에브리타임 홍보했습니다 😎

![KakaoTalk_Photo_2023-09-29-22-16-51](https://github.com/woowacourse-teams/2023-votogether/assets/78212016/300e3350-d3c7-48d7-b3b0-6dbe16e8a385)

**Comment by tjdtls690** [comment]
스레드에서 홍보 해보았습니다!
![캡처](https://github.com/woowacourse-teams/2023-votogether/assets/85877080/f29f1c49-cddb-4db0-aab9-554da9b42e78)

**Comment by jeomxon** [comment]
송파. 강동맘 카페에 글 올렸습니다!
<img width="855" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/51906604/3a3f9eb3-dd5c-41a2-a3db-4e81c4fed545">

**Comment by aiaiaiai1** [comment]
노마드 코더 커뮤니티 사이트
https://nomadcoders.co/community

<img width="1034" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/103165859/625786f8-1e7b-4237-8590-d10ce219f1da">

**Comment by Gilpop8663** [comment]
데빌스플랜 갤러리

https://gall.dcinside.com/mgallery/board/view/?id=devilsplan&no=9688&page=1

![image](https://github.com/woowacourse-teams/2023-votogether/assets/80146176/3e202897-9c0c-47b6-8cda-c58c26985b97)

**Comment by Gilpop8663** [comment]
개발자 취업 정보 공유방 오픈 채팅방 홍보

![image](https://github.com/woowacourse-teams/2023-votogether/assets/80146176/5a3a670c-ce27-4aff-8107-b9d0d6030e8a)

**Comment by Gilpop8663** [comment]
개발자 취업 정보 공유방 오픈 채팅방 홍보 222

![image](https://github.com/woowacourse-teams/2023-votogether/assets/80146176/c94d459e-3d77-4c03-b08a-c713dc230cb2)

**Comment by Gilpop8663** [comment]
우아한테크코스 6기 준비방

<img width="1440" alt="image" src="https://github.com/woowacourse-teams/2023-votogether/assets/80146176/879e1f8a-ddce-487b-80c9-418528b9da52">
